src/Microsoft.ApiDesignGuidelines.Analyzers/Core/ImplementIDisposableCorrectly.cs

```diff
@@ -14,6 +17,9 @@ public abstract class ImplementIDisposableCorrectlyAnalyzer : DiagnosticAnalyzer
     {
```

Comment:
You can make this type abstract and delete the C# and VB derived class stubs that we have.